src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -110,10 +173,16 @@ var Marathon = React.createClass({
       error: function () {
         this.setState({fetchState: States.STATE_ERROR});
       }.bind(this),
-      reset: true,
       success: function () {
+        var state = this.state;
         this.fetchDeployments();
-        this.setState({fetchState: States.STATE_SUCCESS});
+        var activeApp = (state.activeAppId) ?
```

Comment:
In this case that should work! :)
But maybe there is an appId called named '/null'?